src/portfolio/APIClient.ts

```diff
@@ -384,6 +451,28 @@ export async function list(token: string): Promise<PortfolioOverview[]> {
   ];
 }
 
+/**
+ * Gets the data needed for the Back-Testing section of the portfolio details page.
+ *
+ * @param token - Authentication token
+ * @param id - Id of the portfolio for which the Back-Testing information is needed
+ * @param from - UNIX timestamp for the date FROM which the user wants to backtest
+ * @param to - UNIX timestamp for the date UNTIL WHICH the user wants to backtest
+ */
+export async function backtesting(
+  token: string,
+  id: string,
+  from: Date,
+  to: Date
+): Promise<Backtesting> {
+  const response = (await request(
+    token,
+    'GET',
+    `/analytics/backtest/${id}?${from}&${to}`
```

Comment:
Using `from` and `to` like this results in date strings like "Fri Apr 09 2021 13:49:00 GMT+0200 (Central European Summer Time)", which won't work inside an URL. The format "yyyy-mm-dd" should be safe.